vendor/src/github.com/docker/engine-api/types/types.go

```diff
@@ -274,6 +274,27 @@ type ExecStartCheck struct {
 	Tty bool
 }
 
+// HealthcheckResult stores information about a single run of a healthcheck probe
+type HealthcheckResult struct {
+	Start, End time.Time
+	ExitCode   int    // 0=healthy, 1=unhealthy, 2=starting, -1=error running probe
+	Output     string // Output from last check
```

Comment:
I'm sorry to bring this up again: this is wrong. This needs to be a `[]byte` and should allow a more than 255 bytes of output. There is simply no good reason for these kinds of limitations.

There are lots of things that might go into a health check result output that you haven't thought of. This is going to be especially true when compound script checks may want to report on a number of parameters.